lib/v2/brave/latest.js

```diff
@@ -17,20 +17,31 @@ module.exports = async (ctx) => {
         .toArray()
         .map((item) => {
             item = $(item);
-
             const title = item.text();
-            const device = item.parent().find('h2').text();
-            const matchVersion = title.match(/(v[\d.]+)/);
-            const matchDate = title.match(/\((.*?)\)/);
-
-            return {
-                title: `[${device}] ${title}`,
-                link: currentUrl,
-                guid: `${currentUrl}#${device}-${matchVersion?.[1] ?? title}`,
-                description: item.next().html(),
-                pubDate: parseDate(matchDate?.[1].replace(/(st|nd|rd|th)?,/, ''), ['MMMM D YYYY', 'MMM D YYYY']),
-            };
-        });
+
+            if (title.includes('Release Notes')) {
+                const device = item.parent().find('h2').text();
+                const matchVersion = title.match(/(v[\d.]+)/);
+                const matchDate = title.match(/\((.*?)\)/);
+                let description = '';
+                let current = item.next();
+
+                while (current.length && !current.text().includes('Release Notes')) {
+                    description += current.html();
+                    current = current.next();
+                }
```

Comment:
You can use [`nextUntil()`](https://cheerio.js.org/docs/api/classes/Cheerio#nextuntil) to achieve the same result in 1 line.
```js
item.nextUntil('h3[id^="release-notes-"]').toArray().map((item) => $(item).html()).join('');
```